util/build/script/src/cargo_build.rs

```diff
@@ -46,6 +46,7 @@ fn strv_env(command: &mut Command, clean_env: bool, values: &[String], env: &str
         str_env(command, clean_env, None as Option<&String>, env);
     } else {
         let v = values.join(sep);
+        eprintln!("Setting {}=\"{}\"", env, &v);
```

Comment:
Not sure if you intended to leave this here.

Reply:
Yeah, thanks.